spec/rails_helper.rb

```diff
@@ -41,6 +41,7 @@
   exit 1
 end
 
+Rails.application.load_seed # loading seeds
```

Comment:
Doesn't this cause an error the second time you run the suite?

Reply:
Yes, I'm seeing it does in Circle CI but not locally.

Reply:
Should we instead put into the README to run ` bundle exec RAILS_ENV=test rake db:seed`?

Reply:
sure, but `RAILS_ENV=test bin/rails db:seed` is how I'd do it.